cub/cub/warp/warp_exchange.cuh

```diff
@@ -36,11 +36,31 @@
 #include <cub/config.cuh>
 #include <cub/util_ptx.cuh>
 #include <cub/util_type.cuh>
+#include <cub/warp/specializations/warp_exchange_shfl.cuh>
+#include <cub/warp/specializations/warp_exchange_smem.cuh>
 
 
 CUB_NAMESPACE_BEGIN
 
 
+enum WarpExchangeAlgorithm
```

Comment:
I believe we should use 
```suggestion
enum class WarpExchangeAlgorithm
```

Reply:
@miscco https://github.com/NVIDIA/cub/pull/670#discussion_r1271787122